docs/src/main/sphinx/functions/datasketches.rst

```diff
@@ -0,0 +1,39 @@
+======================
+DataSketches Functions
+======================
+DataSketches is a high-performance library of stochastic streaming
+algorithms commonly called ”sketches” in the data sciences. Sketches are
+small, stateful programs that process massive data as a stream and can
+provide approximate answers, with mathematical guarantees, to
+computationally difficult queries orders-of-magnitude faster than
+traditional, exact methods.
+The DataSketches functions allows querying the fast and memory-efficient `Apache
+DataSkecthes <https://datasketches.apache.org/docs/Community/Research.html>`_
+from Trino. Support for `Theta Sketch Framework <https://datasketches.apache.org/docs/Theta/ThetaSketchFramework.html>`_
+is added, specifically :func:`theta_sketch_union` and :func:`theta_sketch_estimate` functions.
+These functions are used in the ``count distinct`` queries using sketches.
+Datasketches can be created using Hive or Pig using respective sketch APIs.
+
+DataSketches functions
+----------------------
+
+.. function:: theta_sketch_union(sketches) -> sketch
```

Comment:
there are two variants of `theta_sketch_union`, one having parameters. The parameters need to be documented.

Reply:
also, should the docs say this is an aggregation function?

Reply:
> `-> sketch`

there is no "sketch" type. should this be "varbinary"?